<launch>
    <arg name="marker_size" default="5" />
    <arg name="max_new_marker_error" default="0.1" /> <!--0.08 -->
    <arg name="max_track_error" default="1" />  <!--0.2 -->
    <arg name="cam_image_topic" default="/euroc3/cam3/image_raw" />
    <arg name="cam_info_topic" default="/euroc3/cam3/info" />
    <arg name="output_frame" default="cam0" />
    <arg name="package_id" default="5" />
    <arg name="package_offset_x" default="-0.08" />
    <arg name="package_offset_y" default="-0.025" />
    <arg name="package_offset_z" default="-0.076" />
    <arg name="namespace" default="euroc3" />

    
    <node name="ar_track_ctl_node" pkg="ar_marker_client" type="ar_track_node_ctl.py" ns="$(arg namespace)">
        <param name="marker_size" value="$(arg marker_size)"/>
        <param name="max_new_marker_error" value="$(arg max_new_marker_error)"/>
        <param name="max_track_error" value="$(arg max_track_error)"/>
        <param name="cam_image_topic" value="$(arg cam_image_topic)"/>
        <param name="cam_info_topic" value="$(arg cam_info_topic)"/>
        <param name="output_frame" value="$(arg output_frame)"/>
        <param name="package_id" value="$(arg package_id)"/>
        <param name="marker_offset_x" value="$(arg package_offset_x)"/>
        <param name="marker_offset_y" value="$(arg package_offset_y)"/>
        <param name="marker_offset_z" value="$(arg package_offset_z)"/>
    </node>
    
</launch>
